### PR TITLE
[OSDOCS#2464]: Adding warning about --force for CLI cluster update

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-2464

Link to docs preview:
https://67438--docspreview.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating-cluster-cli

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by wking — APPROVED** [review]
/lgtm

**Comment by mburke5678** [comment]
One tiny nit. Otherwise LGTM

File: modules/update-upgrading-cli.adoc [hunk 1]
```diff
@@ -103,6 +103,13 @@ $ oc adm upgrade --to=<version> <1>
 <1> `<version>` is the update version that you obtained from the output of the
 `oc adm upgrade` command.
 
++
+
```
[the remark this answers is not in the document]
Comment (obrown1205) [reply modules/update-upgrading-cli.adoc:107]: resolved :)

**Comment by obrown1205** [comment]
Accidentally deleted branch. Restored and renamed. New PR is https://github.com/openshift/openshift-docs/pull/67704 (exact same content. Please let me know once merge review picks this up and I will close this PR in favor of https://github.com/openshift/openshift-docs/pull/67704. Thank you!!

**Pull request closed without merging**